perception/mil_vision/ros_tools/easy_thresh.py

```diff
@@ -65,7 +65,10 @@ def segment(self):
 
         self.is_done = False
         rect = cv2.minAreaRect(np.array([np.array(self.corners)]))
-        box = cv2.cv.BoxPoints(rect)
+        if cv2.__version__[0] == '2':
+            box = cv2.cv.BoxPoints(rect)
+        else:
+            box = cv2.boxPoints(rect)
```

Comment:
No need to support multiple versions. The rest of our code is just compatible with Opencv3 (which this doesn't even check for).